flake.nix

```diff
@@ -0,0 +1,16 @@
+{
+  description = "Flake for openapi2slate";
+
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+  inputs.flake-utils = {
+    url = "github:numtide/flake-utils";
+    inputs.nixpkgs.follows = "nixpkgs";
```

Comment:
What's the point of this line?

`inputs.nixpkgs.follows = "nixpkgs";`

With it, it gives warning and it seems to work without it.
> warning: input 'flake-utils' has an override for a non-existent input 'nixpkgs'

So it could be just `inputs.flake-utils.url = "github:numtide/flake-utils";`

Reply:
Good point, something I had in my flake template for unknown reasons. Removed it.